src/models/device.ts

```diff
@@ -350,6 +360,10 @@ const getDeviceModel = function (
 			);
 		}
 
+		if (options == null) {
```

Comment:
You can remove this now that you have the default above, right?

Reply:
Sorry yes I missed that